src/Microsoft.Identity.Client/MsalUiRequiredException.cs

```diff
@@ -14,6 +15,37 @@ namespace Microsoft.Identity.Client
     /// </summary>
     public class MsalUiRequiredException : MsalServiceException
     {
+        /// <summary>
+        /// Condition can be resolved by user interaction during the interactive authentication flow.
+        /// See https://aka.ms/msal-net-uirequiredexception-classification for details
+        /// </summary>
+        public const string BasicAction = OAuth2SubError.BasicAction;
```

Comment:
Is this the right place for these? Not all the InvalidGrant suberrors can be mitigated by UI. I think these should be in a different context. Perhaps InvalidGrantClassifications ?